build.sbt

```diff
@@ -1,38 +1,34 @@
-val dotty = Build.dotty
-val `dotty-bootstrapped` = Build.`dotty-bootstrapped`
-val `dotty-interfaces` = Build.`dotty-interfaces`
-val `dotty-doc` = Build.`dotty-doc`
-val `dotty-doc-bootstrapped` = Build.`dotty-doc-bootstrapped`
-val `dotty-compiler` = Build.`dotty-compiler`
-val `dotty-compiler-bootstrapped` = Build.`dotty-compiler-bootstrapped`
-val `dotty-library` = Build.`dotty-library`
-val `dotty-library-bootstrapped` = Build.`dotty-library-bootstrapped`
-val `dotty-library-bootstrappedJS` = Build.`dotty-library-bootstrappedJS`
-val `dotty-sbt-bridge` = Build.`dotty-sbt-bridge`
-val `dotty-sbt-bridge-tests` = Build.`dotty-sbt-bridge-tests`
-val `dotty-staging` = Build.`dotty-staging`
-val `dotty-tasty-inspector` = Build.`dotty-tasty-inspector`
-val `dotty-language-server` = Build.`dotty-language-server`
-val `dotty-bench` = Build.`dotty-bench`
-val `dotty-bench-bootstrapped` = Build.`dotty-bench-bootstrapped`
+val scala3 = Build.scala3
+val `scala3-bootstrapped` = Build.`scala3-bootstrapped`
+val `scala3-interfaces` = Build.`scala3-interfaces`
+val `scala3-doc` = Build.`scala3-doc`
+val `scala3-doc-bootstrapped` = Build.`scala3-doc-bootstrapped`
+val `scala3-compiler` = Build.`scala3-compiler`
+val `scala3-compiler-bootstrapped` = Build.`scala3-compiler-bootstrapped`
+val `scala3-library` = Build.`scala3-library`
+val `scala3-library-bootstrapped` = Build.`scala3-library-bootstrapped`
+val `scala3-library-bootstrappedJS` = Build.`scala3-library-bootstrappedJS`
+val `scala3-sbt-bridge` = Build.`scala3-sbt-bridge`
+val `scala3-sbt-bridge-tests` = Build.`scala3-sbt-bridge-tests`
+val `scala3-staging` = Build.`scala3-staging`
+val `scala3-tasty-inspector` = Build.`scala3-tasty-inspector`
+val `scala3-language-server` = Build.`scala3-language-server`
+val `scala3-bench` = Build.`scala3-bench`
+val `scala3-bench-bootstrapped` = Build.`scala3-bench-bootstrapped`
 val `tasty-core` = Build.`tasty-core`
 val `tasty-core-bootstrapped` = Build.`tasty-core-bootstrapped`
 val `tasty-core-scala2` = Build.`tasty-core-scala2`
-val `dotty-tastydoc` = Build.`dotty-tastydoc`
-val `dotty-tastydoc-input` = Build.`dotty-tastydoc-input`
-val `dotty-bench-run` = Build.`dotty-bench-run`
-val `scala-library` = Build.`scala-library`
-val `scala-compiler` = Build.`scala-compiler`
-val `scala-reflect` = Build.`scala-reflect`
-val scalap = Build.scalap
+val `scala3-tastydoc` = Build.`scala3-tastydoc`
+val `scala3-tastydoc-input` = Build.`scala3-tastydoc-input`
+val `scala3-bench-run` = Build.`scala3-bench-run`
 val dist = Build.dist
 val `community-build` = Build.`community-build`
 
 val sjsSandbox = Build.sjsSandbox
 val sjsJUnitTests = Build.sjsJUnitTests
 val sjsCompilerTests = Build.sjsCompilerTests
 
-val `sbt-dotty` = Build.`sbt-dotty`
+val `sbt-scala3` = Build.`sbt-scala3`
```

Comment:
I think we should keep calling it `sbt-dotty`: if we rename it scala-steward won't automatically update projects, and this plugin is temporary until we merge it back into sbt so it's not a big deal if we keep calling it "sbt-dotty". Additionally you mentioned in the PR description:
>  Modify CB projects: update the name of the SBT plugin. Note that if a CB project hard-codes the SBT plugin in project/plugins.sbt, the --addPluginSbtFile=/Users/kmetiuk/Projects/scala3/dotty/community-build/sbt-scala3-sbt will not take effect.

I think this only happens because you renamed the plugin and so you end up with both plugins loaded at the same time. If you keep the same name then the one set in `addPluginSbtFile` will be used because it has a higher version number.

Reply:
We need to rename it since we want to publish under one organisation - org.scala-lang. We don't need the extra complexity of publishing something to ch.epfl.lamp.

Reply:
I think we should keep it in the epfl organization for the reasons I mentioned above, the complexity from that is less than the complexity from the issues well get if we rename it

Reply:
Special-casing our codebase and workflows is a major source of trouble, it should be avoided. I don't think the breakage of automatic PRs is a major problem – PRs will be fixed easily. And you cannot totally rely on automation in migration anyway.

Reply:
The build is already heavily special-cased for `sbt-dotty`: it does not have the same versioning scheme nor the same publishing pipeline. Compared to that, having a different organization for it is negligible.

Reply:
Leaving the name in place turned out to be easier indeed, so I left it.